src/vs/workbench/contrib/notebook/browser/services/notebookServiceImpl.ts

```diff
@@ -101,7 +105,7 @@ export class NotebookProviderInfoStore extends Disposable {
 
 		const builtinProvidersFromCache: Map<string, IDisposable> = new Map();
 		builtins.forEach(builtin => {
-			builtinProvidersFromCache.set(builtin.id, this.add(builtin));
+			builtinProvidersFromCache.set(builtin.id, this.addNotebookInfo(builtin));
```

Comment:
maybe we add some comments there mentioning that builtin providers should register `Editor` themselves.